packages/contracts-bedrock/test/L1/SharedLockbox.t.sol

```diff
@@ -102,6 +115,19 @@ contract SharedLockboxTest is CommonTest {
         assertEq(address(sharedLockbox).balance, _lockboxBalanceBefore - _value);
     }
 
+    function test_authorizePortal_paused_reverts(address _caller, address _portal) public {
```

Comment:
Missing test NATSPEC, the same on the other 2 tests added

Reply:
Fixed https://github.com/defi-wonderland/optimism/pull/145/commits/4bc0d6dd2d25dd56fe90e41594cb503ce34de4bf